k-distribution/tests/regression-new/map-symbolic-tests-haskell/lookup-49-spec.k

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2019 K Team. All Rights Reserved.
+// generated by gen-tests.sh
+
+requires "map-tests.k"
+
+module LOOKUP-49-SPEC
+    imports MAP-TESTS
+
+    claim <k> lookup ( ( (y |-> 4) [ x ] orDefault 0 ) ==K 4 ) => . </k>
+
+endmodule
```

Comment:
If we instead just have `(y |-> 4) [ x] orDefault 0` directly, the output will be exactly `0`, which is a bit stronger of a test than this one.

But if we add the other 4 tests I mentioned then we should be covering the behaviors of orDefault.